source/brailleDisplayDrivers/lilli.py

```diff
@@ -66,31 +69,36 @@ def terminate(self):
 			pass
 		lilliDll.Close408USB()
 
-	def _get_numCells(self):
+	def _get_numCells(self) -> int:
 		return 40
 
 	def _handleKeyPresses(self):
 		while True:
+			key: Optional[int] = None
 			try:
-				key=lilliDll.ReadBuf()
+				# Python 3: review required
+				# The code seems to assume this returns an int.
+				# I haven't confirmed this.

```

Comment:
I think this assumption is safe. By default, every function from a windll type dll returns a long type.